src/sql/workbench/browser/designer/designer.ts

```diff
@@ -693,7 +706,16 @@ export class Designer extends Disposable implements IThemable {
 						resizable: false,
 						isFontIcon: true
 					});
-					deleteRowColumn.onClick((e) => {
+					deleteRowColumn.onClick(async (e) => {
+						if (tableProperties.showRemoveRowConfirmation && tableProperties.removeRowConfirmationMessage) {
```

Comment:
Consider having some error checking somewhere if the user has `showRemoveRowConfirmation` set but not `removeRowConfirmationMessage`

Alternatively you could make these two properties part of a single object `rowConfirmationOptions` that's optional itself, but then these properties are both required inside of it if it's included.

Reply:
I'll have a default remove row confirmation message.